test/cuda-arch/downstream/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(cuda-clang-cpp-user LANGUAGES CXX)
+
+set(ENABLE_CLANG_CUDA On CACHE BOOL "")
```

Comment:
Is it a normal thing to go from CUDA to clang-cuda? Is there a use case for this? Wouldn't you choose one or the other...

Reply:
this is a pathological example to make sure that the way we are passing down variables never overrides user provided variables